### PR TITLE
(#212) Initial Cypress setup

### LINKED ISSUE #212: Configure Cypress
*Description of what is to be accomplished.*
Install Cypress and all required config in the app
**Parent Issue:** #9999
#211 
### Sub-Tasks
- [ ] Create Component
- [ ] Implement Unit Tests for Component

### Notes

*Some complementary notes if necessary:*
* > Here goes a quote from an email
* Here goes whatever useful information can exist…
* More notes, or technical nuances/direction

### PR DESCRIPTION
**Pull request by kate-mashkina**
- installed cypress
- added dependencies and configs
- added example test file
- added analytics test implement
- added scripts
Closes #212

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -93,6 +93,8 @@
     "start": "node scripts/start.js",
     "build": "node scripts/build.js",
     "test": "node scripts/test.js",
```
Comment (arcepaul) [inline package.json:95]: modify this so that it runs through your test:it step (like the other apps)

File: cypress/fixtures/example.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (arcepaul) [inline cypress/fixtures/example.json:1]: can we lose this file?

File: package.json (hunk 1 above)
Comment (kate-mashkina) [reply package.json]: @arcepaul modified!

File: cypress/fixtures/example.json (hunk 2 above)
Comment (kate-mashkina) [reply cypress/fixtures/example.json]: Gone! Also added it to gitignore since it's being added by default when you run cypress in headed mode

**Review by arcepaul — APPROVED** [review]

**Pull request merged**